modules/async_worker.py

```diff
@@ -722,6 +725,8 @@ def callback(step, x0, x, total_steps, y):
             execution_start_time = time.perf_counter()
 
             try:
+                if async_task.last_stop is not False:
+                    fcbh.model_management.interrupt_current_processing()
```

Comment:
fcbh is not working here.

Reply:
Nice catch, thx! This occurrence must have slipped.